### DISCUSSION TITLE
Auto-detect coverage bounding boxes for brain MRI images project

### DISCUSSION THREAD

**Discussion by Nouran-Muhammad** [Emory-HITI/Niffler#386] [Q&A] [answered]
Hi, i have some questions regarding the project statement.
According to my understanding, I have as an input the MRI brain images and the anatomical region of interest, and i will apply an algorithm (conventional image processing or deep learning) to draw bounding boxes around the region of interest.
I have 2 questions
1- regarding the final output, am i going to assess the accuracy of the drawing of the bounding box (did the box cover the intended region well or not)? or am i going to assess whether the region surrounded by the bounding box covers the area needed by the protocol? or both?

2- will i be given annotated data? for example, MRI images with boxes drawn around the region of interest? because regardless of using conventional image processing or deep learning techniques, i will need reference/labelled data for the validation process of my algorithms, so what will the true/labelled data look like?

Thank you

**Comment by pradeeban** [comment, marked as answer]
1- regarding the final output, am i going to assess the accuracy of the drawing of the bounding box (did the box cover the intended region well or not)? or am i going to assess whether the region surrounded by the bounding box covers the area needed by the protocol? or both?

>> The mentor, an MRI Physicist, with the help of fellow radiologists, can establish the ground truth, for the assessments of accuracy.

2- will i be given annotated data? for example, MRI images with boxes drawn around the region of interest? because regardless of using conventional image processing or deep learning techniques, i will need reference/labelled data for the validation process of my algorithms, so what will the true/labelled data look like?

>> Same as above. At least at the latter stages of the development, you will likely have access to such images. But you should also elaborate on fallback plans. What is your testing strategy if no such images are available to you. This is particularly important as an open-source project. As a large healthcare network of Georgia, we can find sufficient MRI images of brain modality, and also get you access to brain MRI images (protected collections) from the cancer imaging archive.
